llm/ggml.go

```diff
@@ -167,6 +167,29 @@ func (kv KV) ContextLength() uint64 {
 	return kv.u64(fmt.Sprintf("%s.context_length", kv.Architecture()))
 }
 
+func (kv KV) Graph(context, batch int) (int64, bool) {
```

Comment:
```suggestion
func (kv KV) GraphEstimate(context, batch int) (int64, bool) {
```

Nit: I usually like the short names but wonder if we'd want to be more specific here